docs/docsite/rst/os_guide/windows_setup.rst

```diff
@@ -92,6 +92,12 @@ WinRM Setup
 ```````````
 You need to configure the WinRM service so that Ansible can connect to it. There are two main components of the WinRM service that govern how Ansible can interface with the Windows host: the ``listener`` and the ``service`` configuration settings.
 
+.. Note:: 
+    The script for setting up this service has been moved to a new `location <https://github.com/oraNod/ansible-documentation/blob/a1133258b90fbbd5e120bb187bcbe9b907e9f3f8/examples/scripts/ConfigureRemotingForAnsible.ps1>`_.
```

Comment:
```suggestion
    The script for setting up this service is `available to download on GitHub <https://raw.githubusercontent.com/ansible/ansible-documentation/ae8772176a5c645655c91328e93196bcf741732d/examples/scripts/ConfigureRemotingForAnsible.ps1>`_.
```

Reply:
The correct url should be to the raw version of the file available from the `ansible/ansible-documentation` repo, not my fork. ;)

Reply:
Additionally I edited the anchor text to avoid a suspicious url. Please refer to https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

Reply:
Oh, I did'nt notice that the url was from a forked repo and not `ansible/ansible-documentation` . Thanks for the insights.